src/web.rs

```diff
@@ -2,11 +2,15 @@
 
 #![allow(clippy::uninlined_format_args)]
 
+use js_sys::Object;
 use raw_window_handle::WebWindowHandle;
 use wasm_bindgen::JsCast;
+use wasm_bindgen::JsValue;
```

Comment:
Could you merge these two includes into one statement? Makes it cleaner.

Reply:
Done.

I guess we don't want to switch to nightly Rustfmt for [`[imports_granularity](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity)`](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity)?